components/sections/supershardians/ShardiansList.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Container, Flex, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
+
+type ShardiansProps = {
+  categories: Array<any>;
+  filteredData: Array<any>;
+  changeCategory: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+const Shardians = ({ categories, filteredData, changeCategory }: ShardiansProps) => {
```

Comment:
Breaks at this resolution
![image](https://user-images.githubusercontent.com/7558499/168416327-be0529f2-fd3c-4869-be34-9c039bb41948.png)

Reply:
Fixed